### PR TITLE
mariadb@10.11 10.11.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  CMake Error at cmake/libfmt.cmake:45 (MESSAGE):
    system libfmt library is not found or unusable
```

**Comment by ZhongRuoyu** [comment]
To be safe, I suggest waiting for upstream maintainers' feedback on the patch first.

**Comment by chenrui333** [comment]
Nice work!

- relates to https://github.com/MariaDB/server/pull/2732

File: Formula/m/mariadb@10.11.rb
Comment (samford) [inline Formula/m/mariadb@10.11.rb:4]:
```suggestion
  url "https://archive.mariadb.org/mariadb-10.11.5/source/mariadb-10.11.5.tar.gz"
```

Comment (ZhongRuoyu) [reply Formula/m/mariadb@10.11.rb]: Applied suggestion.

**Review by iMichka — APPROVED** [review]

**Pull request merged**
